content/getting-started/cost-tools.md

```diff
@@ -38,7 +37,6 @@ Track the min-max of key utilization metrics across your environment.
 
 [1]: /billing-analysis/
 [2]: /right-sizing/reports-cost
-[3]: /right-sizing/reports-asg-recommendations
```

Comment:
Will these numbers show up as is? If so, you should change the 4, 5, 6 below to 3, 4, 5.

Reply:
they won't show up at all I guess, they are just links between the hypertext and the URLs that are at the bottom of the page

Reply:
yup, this is what it looks like in the deploy preview. Links are working fine
<img width="660" alt="Screenshot 2020-12-02 at 15 27 10" src="https://user-images.githubusercontent.com/35566439/100878474-ea05be00-34b2-11eb-98e4-ce95f6bdb48f.png">